CHANGELOG.md

```diff
@@ -52,6 +52,14 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### NI-SCOPE
     * #### Added
+        * `end of acquisition event terminal name` attribute
```

Comment:
those are not the names of the attributes, please write them exactly as they appear on the API

Reply:
You could also add them all in one liner and say something like:

* API parity with NI-SCOPE 20.0
   * `foo`
   * `bar`
   * `baz`